crates/stackable-telemetry/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [99.0.0] - 1970-01-01
```

Comment:
note: This seems to be misplaced. Let's remove it.

Reply:
Gone in ae2a88a